Gopkg.toml

```diff
@@ -22,7 +22,7 @@
 
 [[constraint]]
   name = "github.com/Azure/go-autorest"
-  version = "10.15.3"
```

Comment:
I'd rather update just the lock file, and set the constraint to "anything above this min version we need for the sdk"

Reply:
Derp indeed.  I'll update the constraint to `^10.15.4`.

Reply:
Hmm actually `dep` will assume `^` operator by default.  I think we can merge this as-is but in the future we don't need to update the `.toml` file.